### DISCUSSION TITLE
Show only Change Network button using `ConnectButton`

### DISCUSSION THREAD

**Discussion by estarossa0** [rainbow-me/rainbowkit#1931] [Ideas]
Hello, currently using SIWE and have a custom user button where user login/logout, trying to add switch network option and it would be great if you can just use the `Change network` button provided by RainbowKit without showing the account button at all.

Closest you can get (with `accountStatus="avatar" chainStatus="full" showBalance={false}`) is:
![image](https://github.com/rainbow-me/rainbowkit/assets/57002508/ef5fa995-c12c-4762-8f50-51dc47ce271e)

would be great if you can just the button below with maybe `accountStatus="none"` or maybe export it as it's own component ?
![image](https://github.com/rainbow-me/rainbowkit/assets/57002508/054c69d2-24dc-4341-9600-e1bd42a6eefc)

**Comment by 0xmkh** [comment]
@estarossa0 You want to have the ability to change the network before user connects their wallet ? Is that the case ? 

Also with SIWE i think it makes sense to not show the chain button if you're not signed it yet. This means if you connect your wallet right after you won't be able to switch chain unless you authenticate fully.
